dbt/include/athena/macros/adapters/metadata.sql

```diff
@@ -4,7 +4,7 @@
 
 
 {% macro athena__list_schemas(database) -%}
-  {{ return(adapter.list_schemas()) }}
+  {{ return(adapter.list_schemas(database)) }}
```

Comment:
@Jrmyy yeah, i got the following error:
```
Runtime Error
  Compilation Error in model supply_coordinator_taxon_head_list (models\...\model_script.sql)
    AthenaAdapter.list_schemas() missing 1 required positional argument: 'database'
```

Reply:
What operation are you running to get this ?
I am trying to understand since when we have the issue, and if it is a side effect of the migration to dbt-core 1.5

Reply:
@Jrmyy I tried to execute simple script, and one of the commands was the next string: 
`{%- if execute and (target.schema in list_schemas(target.database)) -%}`